include/aws/common/linked_list.h

```diff
@@ -212,6 +212,34 @@ AWS_STATIC_IMPL void aws_linked_list_insert_after(
     AWS_POSTCONDITION(after->next == to_add);
 }
 
+/**
+ * Swaps the order two nodes in the linked list.
```

Comment:
Something seems weird here.  The postconditions imply a and b must be adjacent with a before b, but the comments don't mention anything about such requirements.   The preconditions don't verify any adjacency.

Reply:
that's left over from previous iteration, willupdate.